test/components/interpolatedColorLegendTests.ts

```diff
@@ -99,6 +102,18 @@ describe("InterpolatedColorLegend", () => {
     svg.remove();
   });
 
+  it("does not crash when font-size is 0px", () => {
+    let legend = new Plottable.Components.InterpolatedColorLegend(colorScale);
+    legend.renderTo(svg);
+    let style = (<any> legend)._element.append("style");
+    style.attr("type", "text/css");
+    style.text(".plottable .legend { font-size: 0px; }");
```

Comment:
This doesn't actually produce the desired effect; you need
```CSS
.plottable .legend text { font-size: 0px; }
```
To be specific enough.